src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -1,12 +1,143 @@
 ---
-title: CLI Reference
+title: CLI Commands
 i18nReady: true
 ---
 import Since from '~/components/Since.astro';
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
 
 
+You can use the Command-Line Interface (CLI) provided by Astro to develop, build, and preview your project from a terminal window.
```

Comment:
The additions to this page are really helpful! Happy to see these get a clear explanation.